### PR TITLE
Restore projects sync

### PR DESCRIPTION
**Pull request by forki**
/cc @enricosada @Krzysztof-Cieslak @alfonsogarciacaro

I wasn't able to test it, but what do you gys think?

### CODE REVIEW

**Comment by forki** [comment]
I assume we still need to await the promise here. How can we do that?

**Comment by forki** [comment]
I guess AwaitPromise should do the trick

File: src/Components/MSBuild.fs [hunk 1]
```diff
@@ -220,6 +220,20 @@ module MSBuild =
         let host = tryGetRightHost' path
         invokeMSBuild path target (Some host)
 
+    let restoreMailBox = MailboxProcessor.Start(fun inbox->
```
Comment (enricosada) [inline src/Components/MSBuild.fs]: MailboxProcessor.Start now work in fable too?

Comment (alfonsogarciacaro) [reply src/Components/MSBuild.fs]: Yes, there's [limited support for MailboxProcessor](http://fable.io/docs/compatibility.html#fsharp-core) (check the caveats)

**Comment by forki** [comment]
Is it released?

**Comment by Krzysztof-Cieslak** [comment]
Yeah, it was released in `3.14.5` - https://github.com/ionide/ionide-vscode-fsharp/blob/master/RELEASE_NOTES.md#3145---15122017

**Pull request merged**